cirq/protocols/__init__.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""  Protocols and methods for quantum channels, unitary and mixture """
```

Comment:
Instead of providing an (incomplete) list of protocols implemented here (which is likely to become stale even if it were complete today), you should instead state what this package contains. Specifically, the package defines *protocols* used by various classes implemented in cirq.

(A protocol is a type defined implicitly by operations you can perform on its instances as opposed to classes which define types explicitly. Such implicit definition of types is called *structural typing*. See [here](https://www.python.org/dev/peps/pep-0544/#terminology) for some background on the concept.)